viewer/packages/api/src/public/migration/Cognite3DViewer.ts

```diff
@@ -1059,31 +1061,65 @@ export class Cognite3DViewer {
    * const url = await viewer.getScreenshot();
    * const image = document.createElement('img');
    * image.src = url;
-   * document.body.appendChild(url);
+   * document.body.appendChild(image);
```

Comment:
Should there be some information about how HTML elements are handled and any corner cases? Maybe also extend documentation site with information and example

Reply:
I added some documentation and one example. 💪